bigquery/storage/managedwriter/connection.go

```diff
@@ -36,6 +37,8 @@ const (
 
 var (
 	errNoRouterForPool = errors.New("no router for connection pool")
+	// TODO: consider if this should be user configurable
```

Comment:
Make sure there's a bug or an issue open for this todo

As for if it should be user configurable, maybe, but it depends on how it would impact other timeouts configured by gax or at the service level

Reply:
Done.